doc/extension_store/README.md

```diff
@@ -0,0 +1,76 @@
+# Proposal for Extension Store Manifest
+
+Example for manifest: [manifest.jsonc](./manifest.jsonc)
+
+## Variable Replacements
+
+* Variables placeholders use the `"My variable 'variableName' has value '${variableName}'"` syntax
+* Naming conventions:
+  * Variables prompted to users are written in `lowerCamelCase`
+  * Global constants defined by the system admin:
+    * `BFS_SERVICE`
+    * `BUCKET`
+* Variables prompted to users are defined in the `parameters` section. The Extension Store asks the user to input values for all parameters.
+* Variable placeholders in strings (e.g. BucketFS paths or SQL scripts) are expanded before using them.
+
+## Parameters to Store
+
+The Extension management component on the cluster must store the following information for an installed VS:
+
+* For each configuration:
+    * ID and version
+        * Required to detect if an older version is already installed
+        * Required to detect if the user tries to install another VS using the same configuration
+    * Paths to installed libraries and configuration files on BucketFS
+        * File names might change in newer versions.
+        * Needed to delete files from BucketFS during update/delete
+    * Parameters with `scope=configuration`
+        * Required when creating a new VS using the same configuration.
+        * Cannot be changed during update. Maybe hide them?
+* For each Connection / Virtual Schema:
+    * ID and version of the configuration
```

Comment:
Splitting into configuration and connection is good, because this we also have in other projects very likely.